ci-operator/config/openshift/release/openshift-release-master__nightly-4.10.yaml

```diff
@@ -362,6 +362,15 @@ tests:
     - chain: ipi-gcp-pre
     - ref: fips-check
     workflow: openshift-e2e-gcp
+- as: e2e-openstack-3nodes
```

Comment:
The convention is to call the 3 nodes cluster where masters are schedulable a compact cluster, so you should rename this job to `e2e-openstack-compact`.

Reply:
done